lambdas/ecs-events-to-slack/notify.py

```diff
@@ -0,0 +1,142 @@
+import json
+import boto3
+import requests
+from dateutil.parser import parse
+from botocore.exceptions import ClientError
+import os
+
+
+def message_formatter(event, region):
+    message_color = "warning"
+
+    event_id = event["id"]
+    event_time = event["time"]
+    parsed_time = parse(event_time)
+    formatted_time = str(parsed_time.replace(microsecond=0))
+
+    event_detail = event["detail"]
+    container_name = event_detail["overrides"]["containerOverrides"][0]["name"]
+    task_arn = event_detail["taskArn"]
+    task_id = task_arn.split("/")[1]
```

Comment:
Any chance of this index being out-of-bounds?

Reply:
It _shouldn't_ be out of bounds unless AWS decides to change the `arn` structure...